app/src/main/java/ar/com/wolox/android/example/ui/training/homePage/PageAdapter.kt

```diff
@@ -0,0 +1,30 @@
+package ar.com.wolox.android.example.ui.training.homePage
+
+import android.support.v4.app.Fragment
+import android.support.v4.app.FragmentManager
+import android.support.v4.app.FragmentPagerAdapter
+
+class PageAdapter(fragmentManager: FragmentManager) : FragmentPagerAdapter(fragmentManager) {
+
+    override fun getItem(position: Int): Fragment {
+        return when (position) {
+            0 -> {
+                NewsFragment()
+            }
+            else -> {
```

Comment:
I don't think it's the best to use the else here because 1. it is really unexpressive and 2. it is not flexible, if you add more pages this will bring you troubles, but I am not sure the best way to do this. I think the best is to throw an exception in the else, but @lmiotti may have another opinion.

Reply:
@dylan-muszel I don't remember well, but 'else' case is an obligation when using 'when'. For that reason, maybe @alejandrobodichon  can use conditional 'if' to return Tab selected.

Reply:
@alejandrobodichon we just talked with @lmiotti and got the conclusion that the best way to make it expressive and flexible is to use the `when` and to throw an exception on the `else`. 😄